logging-backend/src/main/java/io/spine/logging/backend/system/AbstractBackend.java

```diff
@@ -26,6 +37,9 @@
 /**
  * Common backend to handle everything except formatting of log message and metadata. This is an
  * unstable implementation and should not be used outside of the Flogger core library.
+ *
+ * @see <a href="https://github.com/google/flogger/blob/cb9e836a897d36a78309ee8badf5cad4e6a2d3d8/api/src/main/java/com/google/common/flogger/backend/system/AbstractBackend.java">
```

Comment:
Just a thought. We probably need to consider using some URL shortener some day. There was free one from Google, but it's no longer available. I couldn't find a good alternative.

Reply:
@armiol, here's one option, which is free and looks as usable. Please have a look.

Reply:
https://free-url-shortener.rb.gy/